src/compas_fab/robots/robot.py

```diff
@@ -1207,44 +1330,44 @@ def plan_motion(self, goal_constraints, start_configuration=None,
                     max_velocity_scaling_factor=1.,
                     max_acceleration_scaling_factor=1.,
                     attached_collision_meshes=None):
-        """Calculates a motion path.
+        """Calculate a motion path.
 
         Parameters
         ----------
-        goal_constraints: list of :class:`compas_fab.robots.Constraint`
+        goal_constraints : :class:`list` of :class:`Constraint`
             The goal to be achieved, defined in a set of constraints.
             Constraints can be very specific, for example defining value domains
             for each joint, such that the goal configuration is included,
             or defining a volume in space, to which a specific robot link (e.g.
             the end-effector) is required to move to.
-        start_configuration: :class:`compas_fab.robots.Configuration`, optional
+        start_configuration : :class:`Configuration`, optional
             The robot's full configuration, i.e. values for all configurable
             joints of the entire robot, at the starting position. Defaults to
             the all-zero configuration.
-        group: str, optional
+        group : :class:`str`, optional
             The name of the group to plan for. Defaults to the robot's main
             planning group.
-        path_constraints: list of :class:`compas_fab.robots.Constraint`, optional
+        path_constraints : :class:`list` of :class:`Constraint`, optional
             Optional constraints that can be imposed along the solution path.
-            Note that path calculation won't work if the start_configuration
-            violates these constraints. Defaults to None.
-        planner_id: str
-            The name of the algorithm used for path planning. Defaults to 'RRT'.
-        num_planning_attempts: int, optional
+            Note that path calculation won't work if the `start_configuration`
+            violates these constraints. Defaults to ``None``.
+        planner_id : :class:`str`
+            The name of the algorithm used for path planning. Defaults to ``'RRT'``.
+        num_planning_attempts : :class:`int`, optional
             Normally, if one motion plan is needed, one motion plan is computed.
             However, for algorithms that use randomization in their execution
-            (like 'RRT'), it is likely that different planner executions will
+            (like ``'RRT'``), it is likely that different planner executions will
             produce different solutions. Setting this parameter to a value above
             1 will run many additional motion plans, and will report the
             shortest solution as the final result. Defaults to 1.
```

Comment:
Should this "1" and the following default values be surrounded by ticks?

Reply:
8616817